app/reaction.go

```diff
@@ -136,10 +136,4 @@ func (a *App) sendReactionEvent(event string, reaction *model.Reaction, post *mo
 
 	post.HasReactions = hasReactions
 	post.UpdateAt = model.GetMillis()
-
```

Comment:
If lines below are removed, 2 lines above can be removed as well.

Reply:
You are right. Will clean it up